modules/k8s_ingress/variables.tf

```diff
@@ -65,6 +65,11 @@ variable "enable_whitelist_ip" {
   default = false
 }
 
+variable "is_kong_auth_header_block_public_access" {
+  type = bool
+  default = true
```

Comment:
I know. it is a small thing, but... could you use fmt for the code.
I believe the formatting code can be easy achievable via executing the cli command `terraform fmt`, or via `IDE plugins`.